Mustachio/Parser.cs

```diff
@@ -143,10 +143,7 @@ public static ExtendedParseInformation ParseWithModelInference(string templateSo
                         buildArray.Add(HandleSingleValue(currentToken, options, currentScope));
                         break;
                     case TokenType.Custom:
-                        if (currentToken.Renderer != null)
-                        {
-                            buildArray.Add(currentToken.Renderer(currentToken.Value, tokens, options, currentScope));
-                        }
+                        buildArray.Add(currentToken.Renderer(currentToken.Value, tokens, options, currentScope));
                         break;
```

Comment:
This check is no longer needed here since we do it in the Tokenizer. (I like it more now)